dashboard/lib/pd/foorm/survey_reporter.rb

```diff
@@ -112,13 +117,17 @@ def self.get_facilitator_rollup_for_course(facilitators, course_name, rollup_que
     # If split_by_facilitator is true, split questions by facilitator id.
     def self.get_rollup_for_workshop_ids(
       workshop_ids,
-      rollup_question_details,
+      questions_to_summarize,
       split_by_facilitator,
       facilitators,
       facilitator_id=nil
     )
       ws_submissions, form_submissions, forms = get_raw_data_for_workshop(workshop_ids, facilitator_id)
-      _, summarized_answers = parse_and_summarize_forms(ws_submissions, form_submissions, forms)
+      parsed_forms, summarized_answers = parse_and_summarize_forms(ws_submissions, form_submissions, forms)
+      rollup_question_details = Pd::Foorm::RollupHelper.get_question_details_for_rollup(
```

Comment:
moved this calculation here because if a facilitator has facilitated both CSF Intro and CSF Deep Dive the rollup should include questions from both post-surveys, where before we would only look for the post-survey of the workshop requesting the rollup.